<!--AUTO-GENERATED PUBLISH JOB COMMENT-->
# 🤖 AutoRest automatic publish job 🤖
## failed
~~~ Haskell
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.

> dotnet-sdk-2.0.0@1.4.4 postinstall C:\Users\ci\AppData\Local\Temp\PUBLISH8fzkm\29_20180201T185248\autorest.common\node_modules\dotnet-sdk-2.0.0
> node -e "/*PostInstall: Installs platform-specific .NET framework */try{require('./dist/app.js')}catch(e){}"


> @microsoft.azure/autorest.common@2.4.0 postinstall C:\Users\ci\AppData\Local\Temp\PUBLISH8fzkm\29_20180201T185248\autorest.common
> gulp restore

[10:53:53] Using gulpfile ~\AppData\Local\Temp\PUBLISH8fzkm\29_20180201T185248\autorest.common\gulpfile.js
[10:53:53] Starting 'init'...
[10:53:53] Finished 'init' after 168 μs
[10:53:53] Starting 'restore'...
           C:\Users\ci\AppData\Local\Temp\PUBLISH8fzkm\29_20180201T185248\autorest.common :: dotnet restore C:\Users\ci\AppData\Local\Temp\PUBLISH8fzkm\29_20180201T185248\autorest.common\src\autorest.common.csproj /nologo
           C:\Users\ci\AppData\Local\Temp\PUBLISH8fzkm\29_20180201T185248\autorest.common :: dotnet restore C:\Users\ci\AppData\Local\Temp\PUBLISH8fzkm\29_20180201T185248\autorest.common\test\autorest.common.test.csproj /nologo
[10:54:00] Finished 'restore' after 6.63 s
npm notice created a lockfile as package-lock.json. You should commit this file.
added 385 packages in 69.429s

> @microsoft.azure/autorest.common@2.4.0 publish-preview C:\Users\ci\AppData\Local\Temp\PUBLISH8fzkm\29_20180201T185248\autorest.common
> NOT YET AUTOMATED

'NOT' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @microsoft.azure/autorest.common@2.4.0 publish-preview: `NOT YET AUTOMATED`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @microsoft.azure/autorest.common@2.4.0 publish-preview script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ci\AppData\Roaming\npm-cache\_logs\2018-02-01T18_54_02_241Z-debug.log

~~~

--- a/src/autorest.common.csproj
+++ b/src/autorest.common.csproj
@@ -5,8 +5,8 @@
     <PackageId>Microsoft.AutoRest.Common</PackageId>
     <AssemblyName>Microsoft.AutoRest.Common</AssemblyName>
     <AssemnblyTitle>Microsoft AutoRest Common</AssemnblyTitle>
-    <Version>2.4.37</Version>
-    <Description>PLEASE PROVIDE DESCRIPTION about your pacakge</Description>
+    <Version>1.0.0</Version>
+    <Description>Common code shared between AutoRest modeler and code generators</Description>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <Authors>Microsoft</Authors>
     <Owners>Microsoft, azure-sdk</Owners>
@@ -18,11 +18,9 @@
     <PackageTags>autorest.common;AutoRest</PackageTags>
     <PacakgeReleaseNotes>
       <![CDATA[
-        1. Current Release specific notes
+        1. Provides common code
       ]]>
     </PacakgeReleaseNotes>
-  </PropertyGroup>
-  <PropertyGroup>
     <GenerateFullPaths>true</GenerateFullPaths>
     <Common>$(MsBuildThisFileDirectory)</Common>
     <SolutionDir>$(Common)../</SolutionDir> 

